rust/toodle_ffi/src/lib.rs

```diff
@@ -0,0 +1,321 @@
+extern crate ffi_utils;
+extern crate libc;
+extern crate mentat_core;
+extern crate time;
+extern crate toodle;
+
+mod ctypes;
+
+use libc::{ c_int, size_t, time_t };
+use mentat_core::Uuid;
```

Comment:
We should make sure that `Uuid` is re-exported from `mentat`, perhaps as `mentat::types::{Uuid, OrderedFloat, …}`, so you don't need to depend on an internal crate.